### PR TITLE
hw10.1

### PR DESCRIPTION
**Pull request by Mykhaylo12**

### CODE REVIEW

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 1]
```diff
@@ -17,7 +23,19 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        String fileText = Files.readString(Paths.get(fileName));
+        StringBuilder sb = new StringBuilder();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: `sb` isn't informative name of the variable

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 2]
```diff
@@ -17,7 +23,19 @@
...
+        TreeSet<String> text = new TreeSet<>(Arrays.asList(fileText.replaceAll("[^a-zA-Z]", "")
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: better using Character than String, and methods Character.isLetter(), Character.toLowerCase()

Comment (Sofasmile) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]:
```suggestion
        Set<String> text = new TreeSet<>(Arrays.asList(fileText.replaceAll("[^a-zA-Z]", "")
```

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 3]
```diff
@@ -17,7 +23,19 @@
...
+                .toLowerCase().split("")));
+        for (String i : text) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: i -?

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 4]
```diff
@@ -17,7 +23,19 @@
...
+            sb.append(i);
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: if(sb.length >= 5){
break;
}

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 1 above)
Comment (Mykhaylo12) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: I will call stringBuilder

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 2 above)
Comment (Mykhaylo12) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: ok

Comment (Mykhaylo12) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: ok

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 3 above)
Comment (Mykhaylo12) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: i will change

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 4 above)
Comment (Mykhaylo12) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: have common sense

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**